PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,35 +19,44 @@
         </activity>
         <activity
             android:name=".GameActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".AvatarRoomActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".AvatarActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".MapActivity"
             android:label="@string/app_name"
-            android:launchMode="singleTop" />
+            android:launchMode="singleTop"
+            android:theme="@style/MyNoActionBar"/>
```

Comment:
@chhavip I created an abstract theme for all the activities so that it is easier to incorporate any changes in theme design in future.